### PR TITLE
[BH-1264] Add alarm sound fade in

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
Adding a fade in for the alarm sound (main and power nap) according to the pattern (V): dt = 30[s] * V/10.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

**Review by lukasmas — APPROVED** [review]

File: products/BellHybrid/alarms/src/actions/PlayAudioActions.cpp [hunk 1]
```diff
@@ -24,8 +24,15 @@ namespace alarms
         if (duration) {
             spawnTimer(*duration);
         }
-        return service.bus.sendUnicast(std::make_shared<service::AudioStartPlaybackRequest>(path, playbackType),
-                                       service::audioServiceName);
+        const auto fadeInSettings = utils::getNumericValue<bool>(
+            settings.getValue(bell::settings::Alarm::fadeActive, settings::SettingsScope::Global));
+        const auto fadeInEnabled =
+            fadeInSettings
```
Comment (Lefucjusz) [inline products/BellHybrid/alarms/src/actions/PlayAudioActions.cpp:30]: This nested ternary condition is IMO quite hard to read, if I understand correctly it should be equivalent to 

```
const auto fadeInEnabled = (fadeInSettings && (playbackType == audio::PlaybackType::Alarm)) ? audio::FadeIn::Enable : audio::FadeIn::Disable;
```

**Review by Lefucjusz — APPROVED** [review]
Looks good! Please check two minor comments I've left

File: products/BellHybrid/services/audio/include/audio/VolumeFadeIn.hpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+#include <Timers/TimerFactory.hpp>
+#include <Audio/AudioCommon.hpp>
+
+namespace audio
+{
+    class VolumeFadeIn
+    {
+      public:
+        using setCallback = std::function<void(float)>;
```
[the remark this answers is not in the document]
Comment (Maciej-Mudita) [reply products/BellHybrid/services/audio/include/audio/VolumeFadeIn.hpp]: done

File: products/BellHybrid/alarms/src/actions/PlayAudioActions.cpp (hunk 1 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/alarms/src/actions/PlayAudioActions.cpp]: done

**Pull request merged**